src/import/product-import.service.ts

```diff
@@ -76,6 +78,14 @@ export class ProductImportService {
   }
 
   private async productImport(period?: number) {
+    const meatdataSchemas = await this.voucherifyClient
+      .getClient()
+      .metadataSchemas.list();
+    const metadataSchema = meatdataSchemas.schemas.find(
+      (schema) => schema.related_object === 'product',
+    );
+    const metadataSchemaProperties = Object.keys(metadataSchema.properties);
+
```

Comment:
3x the same logic with different schema.related_object variable. Consider wrapping it in the same method e.g in VoucherifyConnectorService.